content/en/docs/concepts/traffic-management/index.md

```diff
@@ -254,502 +173,243 @@ spec:
-kind: VirtualService
-metadata:
-  name: my-vtl-svc
-spec:
-  hosts:
-  - "*.my-co.org"
-  http:
-  - route:
-    - destination:
-        host: my-svc
-        subset: v1
+hosts:
+- reviews
 {{< /text >}}
 
-In the example, under `spec`,
-`hosts` lists the virtual service's hosts. In this case, the
-hosts are `*.my-co.org`, where `*` is a wildcard prefix indicating that this
-virtual service handles routing for any DNS name ending with `.my-co.org`.
-
-You can specify user-addressable hosts by using any DNS name or an internal
-mesh service name as long as the name resolves, implicitly or explicitly, to
-one or more fully qualified domain names (FQDN). To specify multiple hosts, you
-can use wildcards.
-
-Also, note that under `route`, which specifies the routing rule's
-configuration, and `destination`, which specifies the routing rule's
-destination, `host: my-svc` specifies the destination's host. If you are
-running on Kubernetes, then `my-svc` is the name of a Kubernetes service.
-
-You use the destination's host to specify where you want the traffic to be
-sent. The destination's host must exist in the service registry. To use
-external services as destinations, use [service entries](/docs/concepts/traffic-management/#service-entries)
-to add those services to the registry.
-
-{{< warning >}}
-Istio **doesn't** provide [DNS](https://hosting.review/web-hosting-glossary/#9)
-resolution. Applications can try to resolve the FQDN by using the DNS service
-present in their platform of choice, for example `kube-dns`.
-{{< /warning >}}
-
-The following diagram shows the configured rule:
-
-{{< image width="40%"
-  link="./virtual-services-4.svg"
-  caption="Configurable traffic route to send traffic to a specific subset"
-    >}}
-
-### Route requests to services in a Kubernetes namespace {#routing-namespace}
-
-When you specify the `host` field for the destination of a route in a virtual service
-using a short name like `svc-1`, Istio expands the short name into a fully qualified domain name.
-To perform the expansion, Istio adds a domain suffix based on the namespace of the virtual service that
-contains the routing rule. For example, if the virtual service is defined in the `my-namespace` namespace,
-Istio adds the `my-namespace.svc.cluster.local` suffix to the abbreviated destination resulting in
-the actual destination: `svc-1.my-namespace.svc.cluster.local`.
-
-While this approach is very convenient and commonly used to simplify examples, it can
-easily lead to misconfigurations. Therefore we do
-[not recommend it for production deployments](/docs/reference/config/networking/v1alpha3/virtual-service/#Destination).
-
-The following example shows a virtual service configuration with fully qualified traffic routes
-for two services in the `my-namespace` Kubernetes namespace.
-The configuration relies on the URI prefixes of the two services to distinguish
-them.
+The virtual service hostname can be a DNS name or an internal mesh service name
```

Comment:
I don't really understand this sentence, so others may find it confusing as well.

Reply:
Let me see if I can clarify it a bit. PTAL!